examples/client/tui/commands.nim

```diff
@@ -722,6 +722,72 @@ proc command*(self: Tui, command: SendMessage) {.async, gcsafe, nimcall.} =
   else:
     asyncSpawn self.client.sendMessage(command.message)
 
+# Call ----------------------------------------------------------------
+
+proc help*(T: type CallRPC): HelpText =
+  let command = "call"
+  HelpText(command: command, aliases: aliased[command], parameters: @[
+    CommandParameter(name: "input", description: "Options JSON")
+    ], description: "Calls an Ethereum RPC method")
+
+proc new*(T: type CallRPC, args: varargs[string]): T =
+  var
+    input = ""
+  if args.len > 0:
+    input = args[0]
+  T(input: input)
```

Comment:
Maybe we could have `input` be `JsonNode`, such that we are forced to validate correct JSON input here. Especially if the user is typing in JSON 😳  If the JSON can't be parsed or is invalid, it would prevent a round-trip through the task runner.